src/main/resources/lang/default.lang

```diff
@@ -491,6 +493,15 @@ entities:
 	dark oak boat:
 		name: dark oak boat¦s
 		pattern: dark oak boat(|1¦s)
+	mangrove boat:
+		name: mangrove boat¦s
+		pattern: mangrove boat(|1¦s)
+	bamboo boat:
```

Comment:
I know this is referred to as 'bamboo raft' in some places - do we need both variations here?

Reply:
For the aliases yes. The id is minecraft:bamboo_raft
This does not matter what the name is. This is used by the EntityData class.

I will update it to raft though.

Reply:
Again recoding the EntityData for BoatData eventually. Currently Bamboo and Cherry boats aren't released, as it's 1.17+ as mentioned above.